buf.gen.yaml

```diff
@@ -0,0 +1,16 @@
+version: v1beta1
```

Comment:
This is totally an aesthetic nitpick but is there any way to get these config files out of the root of the project to reduce the amount of config pollution there?

Reply:
It is possible to tell `buf` where to get the files via the `--template` and `--config` flags; I can update it. What's a reasonable subdirectory for this? `.buf`?

Reply:
That works. Or somewhere in the existing `tools/` dir